tests/runner.py

```diff
@@ -1121,9 +1123,8 @@ def get_poppler_library(self, env_init=None):
   def get_zlib_library(self):
     if WINDOWS:
       return self.get_library(os.path.join('third_party', 'zlib'), os.path.join('libz.a'),
-                              configure=[path_from_root('emconfigure.bat')],
-                              configure_args=['cmake', '.'],
-                              make=['mingw32-make'],
+                              configure=['cmake', '.'],
+                              make=['cmake', '--build', '.'],
```

Comment:
Is this change intended? (Before it was mingw..)

Reply:
Yes, its actually better to call `cmake --build` because then it will invoke what ever the chosen build system was.   Calling `mingw32-make` is more fragile, and in the case the `mingw32-make` was actually used by cmake it is equivalent.